hyperon_das/factory.py

```diff
@@ -5,10 +5,13 @@
 from hyperon_das_atomdb import IAtomDB
 from hyperon_das_atomdb.adapters import InMemoryDB, RedisMongoDB
 
+from tests.mock import DatabaseMock
```

Comment:
Importing unit test stuff into public components is a bad design. Perhaps you should extend `DatabaseFactory` inside the `test` package? Then add the methods/enums you need there.

Reply:
Yes, I got it. I removed this 'test' type and create a `DistributedAtomSpaceMock` extended the `DistributedAtomSpace`. I think that is a best approach